scripts/up.sh

```diff
@@ -1,79 +1,79 @@
 #!/usr/bin/env bash
-#
+
 # Bring up the SDK test environment.
 
 set -e
 
-rootdir=`dirname $0`
-pushd $rootdir/.. > /dev/null
+START=$(date "+%s")
 
 # Defaults
 TYPE_OVERRIDE=""
-DAEMON_FLAG="-d"
```

Comment:
Removing most of the interactively configurable functionality of `up.sh` (and `down.sh` as well). I suspect -with no evidence- that this was rarely used. Most of the safe functionality can be achieved by setting variables in `.env`. If there is a specific capability that is lost due to this change and should be kept, please provide feedback.  We can discuss how to keep the functionality- whether by exposing int `.env` or bringing back more interactivity in this script.

@winder @michaeldiamant

Reply:
I suspect these options mainly for when I was developing these fixtures. As long as it still works, I think you should go ahead and make the scripts work for you.